builds/wasm/src/vade/vade-api-bbs.ts

```diff
@@ -128,32 +127,23 @@ class VadeApiBbs extends VadeApiShared {
   /**
    * creates a new keypair for BBS+ based proofs
    *
-   * @param params.keyOwnerDid did that will receive new public key
    * @param options.identity Substrate identity to use (e.g. did:evan:0x12345)
    * @param options.signingKey key to request the signing endpoint with
    */
   public async createBbsKeys(
```

Comment:
Maybe we can put this in one line now. Sorry, cannot suggest this in here due to a deleted line in the area to suggest a change for. oO

```ts
  public async createBbsKeys(options: VadeOptions): Promise<BbsKeys> {
```